### PR TITLE
Minor text adjustments and standardization

### PR DESCRIPTION
**Pull request by snastase**
I did a full pass through the existing text and tried to make the language a little conversational as I went (e.g., less passive tone, use contractions). Where possible, I tried to remove what I considered superfluous words/text in order to make it concise as possible. I think it'll be important to keep the text as tight and punchy as possible, because some people going through this will not actually _want_ to spend any extra time on it. I also tried to make syntactic and punctuation choices consistent throughout—keeping everything well-structured and professional-looking will add to perceived credibility. There’s still seems to be some inconsistencies and confusion about structural elements; e.g., what should be a `callout` vs. a `challenge` vs. a `solution`, when do we call it an “Exercise”, do we use the term “Answer” or “Solution”, when describing exercise do we use the “Goal: X” syntax, etc. I’ll try to resolve these in upcoming commits (and also make suggestions for TODO exercises). Will follow up with additional notes.

### CODE REVIEW

**Comment by snastase** [comment]
Couple additional notes on lesson 01: You mention "POSIX" and "POSIX-compliant" a couple times without defining it. Maybe just a link to the [wikipedia page](https://en.wikipedia.org/wiki/POSIX) would be sufficient? In the shebang example, your script `blah` still needs a `chmod +x` to run with `./blah`, correct? This makes me wonder if it's worth touching on permissions / `chmod` in this lesson... Not sure whether that directly relates to _reproducibility_ per se, but it is an issue people run into a lot. In the discussion of `which`, you mention possible confusion with `locate`, but my usual confusion is between `which` and `whereis` (e.g., this [superuser QA](https://superuser.com/a/40304)). I also wonder if it's worth having further guidance on configuration files, e.g., `.bashrc`, `bash_profile`, etc. People tend to screw these up, certain software (e.g., Anaconda, FreeSurfer?) like to modify them, and some common neuroimaging packages have their own (e.g., `.afnirc`). Let me know what you think and I can try to implement any changes.

**Comment by yarikoptic** [comment]
I am really sorry I had missed your PR @snastase ! Would you have a moment to refresh it and resolve some conflicts which came about since then?  Sorry again for letting it slip

**Comment by snastase** [comment]
@yarikoptic I screwed up the merge process a bit by renaming `00-Overview.md` to `00-overview.md` (for consistency with other filenames), but I did a visual check and I think it should be fine. As far as I can tell, any conflicts in the other files should be straightened out—i.e. switched to your newer version, but with any minor text fixes from my branch.

File: _episodes/01-shell-basics.md [hunk 1]
```diff
@@ -93,33 +89,34 @@ failed interim execution.
...
 
-- **sh** - a POSIX compliant shell. This is a generic name and doesn’t refer to a specific project.
-   - Most portable shell (since it is the standard)
+- **sh** - a POSIX-compliant shell; this is a generic name and doesn’t refer to a specific project.
+   - Most portable shell (since it's the standard)
    - Many FSL scripts use this shell
-- **ksh** - KornSHell. Based on older bash, but also became a root for tcsh, zsh and others
-- **dash** - an implementation of a POSIX compliant shell (**sh**)
+- **ksh** - KornSHell; based on older bash, but also became a root for tcsh, zsh and others.
+- **dash** - Debian Almquist SHell; an implementation of a POSIX-compliant shell (**sh**).
```
Comment (yarikoptic) [inline _episodes/01-shell-basics.md:109]: d'oh - I had no clue about provenance of the name ;-) thanks!

File: _episodes/01-shell-basics.md [hunk 2]
```diff
@@ -596,19 +595,19 @@ containing an empty string:
...
 which can lead to many undesired and non-reproducible
-or highly undesired side-effects:
+or undesirable side-effects:
 
 - "using" mistyped variable names
 - "using" variables that were not defined yet due to the logic
-  of the script.  E.g. imagine effects of `sudo rm -rf ${PREFIX}/` if the
-  `PREFIX` variable was not defined for some reason.
+  of the script. For example, imagine the effects of `sudo rm -rf ${PREFIX}/` if the
+  `PREFIX` variable was not defined for some reason. (**Do not copy this into your terminal!**)
```
Comment (yarikoptic) [inline _episodes/01-shell-basics.md]: ;-)

**Comment by yarikoptic** [comment]
Thank you @snastase ... I ran out of time -- but I did tune it up so diff now is reviewable.  It needs a bottom-up pass to possibly catch remaining typos and gotchas and watch to not reintroduce trailing whitespaces ;)

**Comment by snastase** [comment]
> Thank you @snastase ... I ran out of time -- but I did tune it up so diff now is reviewable. It needs a bottom-up pass to possibly catch remaining typos and gotchas and watch to not reintroduce trailing whitespaces ;)

Should be able wrap it up tomorrow!

**Comment by yarikoptic** [comment]
Awesome, thanks in advance!

**Comment by snastase** [comment]
Okay, pulled your updates and made another pass through this. Had some issues rendering via Jekyll, but I did a close read through the Markdown files. Mostly just adjusting some wording, checking for typos, trying to standardize punctuation/formatting.

File: _episodes/11-wrap-up.md [hunk 3]
```diff
@@ -7,9 +7,9 @@ questions:
...
 - "Recap what you've learned"
 keypoints:
-- “Your own summary of everything so far"
+- “Push yourself to incorporate these practices into your daily workflow."
 ---
 
 ## Sum everything up for yourself.
 
-You can mention specific external resources that were helpful, or particular exercises that did (or didn’t) help you understand a concept. As previously mentioned, you will best learn these practices by doing, and by cataloguing your hands-on experiences with reproducible practices, it may help you permanently adopt these practices in your research.
+You can mention specific external resources that were helpful, or particular exercises that did (or didn’t) help you understand a concept. As previously mentioned, the best way to learn these practices is by **doing** (second only to teaching others!). Keeping a record of your hands-on experiences with reproducible practices will help you permanently adopt these practices in your research.
```
Comment (yarikoptic) [inline _episodes/11-wrap-up.md:15]: So true about the second only ;-)

**Review by yarikoptic — APPROVED** [review]
Skimmed through, looks great, thank you so much Sam! I will assume that the rendering gotchas you had mentioned were resolved and merge (if not - let me know what they were and I will fix them after)

Thanks again

**Pull request merged**